exercises/generics/generics2.rs

```diff
@@ -6,14 +6,12 @@
 // Execute `rustlings hint generics2` or use the `hint` watch subcommand for a
 // hint.
 
-// I AM NOT DONE
-
-struct Wrapper {
-    value: u32,
+struct Wrapper<T: PartialOrd> {
```

Comment:
This is incorrect!
Why the `: PartialOrd`?
You were asked to support wrapping ANY type!

Reply:
changed. It works because both regular integers and &str implement this trait